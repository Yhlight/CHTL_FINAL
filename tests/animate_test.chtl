html {
    body {
        div {
            id: "animated-box";
            style {
                width: 100px;
                height: 100px;
                background-color: blue;
            }
        }

        script {
            animate {
                target: {{#animated-box}},
                duration: 2000,
                easing: "ease-in-out",
                loop: 2,
                direction: "alternate",

                begin: {
                    background-color: "blue",
                    transform: "scale(1)"
                },

                end: {
                    background-color: "red",
                    transform: "scale(1.5)"
                }
            }
        }
    }
}
