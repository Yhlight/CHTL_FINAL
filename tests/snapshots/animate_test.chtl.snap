Successfully generated output.html
Successfully generated output.js
